src/cf-base.less

```diff
@@ -140,70 +140,82 @@ body {
     line-height: @base-line-height;
 }
 
-.heading-1( @fs: 34px, @line-height: 44px ) {
+.heading-1( @fs: 34px ) {
     @font-size: @fs;
     .webfont-regular();
-    // Spacing in the design manual is based off of the baseline so this margin
-    // number has been eyeballed to be 30px.
-    margin-bottom: unit( 16px / @font-size, em );
+
+    margin-bottom: unit( 20px / @font-size, em );
     font-size: unit( @font-size / @base-font-size-px, em );
-    line-height: unit( @line-height / @font-size );
+    line-height: 1.25;
 }
 
-.heading-2( @fs: 26px, @line-height: 33px ) {
+.heading-2( @fs: 26px ) {
     @font-size: @fs;
     .webfont-regular();
-    // Spacing in the design manual is based off of the baseline so this margin
-    // number has been eyeballed to be 30px.
-    margin-bottom: unit( 19px / @font-size, em );
+
+    margin-bottom: unit( 20px / @font-size, em );
     font-size: unit( @font-size / @base-font-size-px, em );
-    line-height: unit( @line-height / @font-size );
+    line-height: 1.25;
+
+    p ~ & {
```

Comment:
Wait, that still won't work because there are a lot of situations, like

<img width="772" alt="screen shot 2015-10-30 at 1 26 11 pm" src="https://cloud.githubusercontent.com/assets/1280430/10852918/d193c7f4-7f09-11e5-9f5f-bb18d7c88e49.png">

Where we don't want the top margin. I think we really do need to scope it with a wrapper class.